### ISSUE TITLE
build/ folder should not be versionned

### ISSUE THREAD

**Issue by Mcdostone** [mxm-tr/travel_history_parser#4]
the `build/` folder should not be versionned:
 - `git rm -r build/`
 - `echo "build/" >> .gitignore`

Oh wait, I figured out. I did version this folder for the [hosting](https://mxm-tr.github.io/travel_history_parser/) right?

**Comment by mxm-tr** [comment]
Yes indeed, I'm using the gh-pages npm script that requires the build folder to be included.

If you know how to make this work without having to version the build that'd be awesome!
